### DISCUSSION TITLE
"DDoS reflection/ amplification attacks against third parties on a daily basis" reported by VPS

### DISCUSSION THREAD

**Discussion by iszhi** [AdguardTeam/AdGuardHome#6393] [Q&A]
Hello, Today I received the security warnings from my VPS provider. They told me that my VPS\s DNS is under attacking by something like: "open DNS resolvers are abused for conducting DDoS reflection/
amplification attacks against third parties on a daily basis.

Please find below a list of open DNS resolvers hosted on
your network which can be abused for DDoS reflection/amplification
attacks if no countermeasures have been implemented. The timestamp
indicates when the open resolver was identified."

What should I do? Does it mean I may lost data?

**Comment by snapsl** [comment]
Please, do not host a public available service if you don't know how to secure it. In general don't host publicly open dns resolvers.

**Reply by iszhi** [reply]
That is why I come here to ask questions.

**Comment by snapsl** [comment]
> What should I do? 

Don't host Adguard Home publicly.

> Does it mean I may lost data?

Your adguard home instance was used to attack other services. Since you hosted it you can be held responsible depending on the legislation of your country.
